### ISSUE TITLE
Profile shifted after converting.

### ISSUE THREAD

**Issue by Mikey-bu** [shtirlitsDva/ProfileToolBox#1]
After converting the polyline to profile, the profile is shifted compared to the polyline. Could there be any reason for this? It is easy enough to shift, but still strange

**Comment by shtirlitsDva** [comment]
Yes, it is strange. It works fine in our office. I cannot think of a cause off the top of my head. Maybe something to do with profile view style or properties.

**Comment by Civil4D** [comment]
This tool is very useful, thank you for providing it.

I believe this issue has to do with the alignment starting station. I have an alignment we are starting at 10+00, but the converted profile is drawing 1000 to the left of where it should be - as if it were being drawn starting at 0+00.

If I change the alignment to start at 0+00 instead, the polyline is converted in the appropriate location in the profile view.

I hope this is helpful.
